index.html

```diff
@@ -2142,11 +2139,12 @@ <h3>Retrieve Verification Method</h3>
 <a href="#INVALID_VERIFICATION_METHOD">INVALID_VERIFICATION_METHOD</a>.
           </li>
           <li>
-If <var>verificationMethod</var> is not associated with the array of
-<var>vmPurposes</var> in the <var>controllerDocument</var>, either by reference
-(URL) or by value (object), an error MUST be raised and SHOULD convey an error
-type of <a href="#INVALID_PROOF_PURPOSE_FOR_VERIFICATION_METHOD">
-INVALID_PROOF_PURPOSE_FOR_VERIFICATION_METHOD</a>.
+If <var>verificationMethod</var> is not associated with the
+<var>verificationRelationship</var> array in the <var>controllerDocument</var>,
+either by reference (URL) or by value (object), an error MUST be raised and
```

Comment:
A little wordy, but I think this might clarify that `verificationRelationship` is used to identify the property of the controller document.

```suggestion
If <var>verificationMethod</var> is not associated, either by
reference (URL) or by value (object), with the array value of the
<var>controllerDocument</var> property identified by
<var>verificationRelationship</var>, an error MUST be raised and
```

Reply:
I applied a change suggested by @selfissued in https://github.com/w3c/controller-document/pull/41/commits/09ab30030b0fa1f3f2db5defd66a962e4522426d ... it's not as precise, but probably gets the concept across. We can do further editorial improvements in future PRs, if necessary.